tests/platform/document/index.html

```diff
@@ -432,9 +432,13 @@ <h1 class="title">Document</h1>
         <script type="text/partytown">
           (function () {
             const doc = document.implementation.createHTMLDocument();
-            doc.body.textContent = '88mph ' + doc.visibilityState;
 
             const elm = document.getElementById('testCreateHTMLDocument');
+
+            const addlElm = doc.createElement('base');
+            const addlElmType = typeof addlElm;
+
+            doc.body.textContent = '88mph ' + doc.visibilityState + ' ' + addlElmType;
```

Comment:
@adamdbradley this is where I added the test from [#174 Comment](https://github.com/BuilderIO/partytown/issues/174#issuecomment-1121708179)